apps/range-voting/app/components/ProgressBar.js

```diff
@@ -26,13 +26,24 @@ const Main = styled.div`
   width: 100%;
   align-items: center;
 `
+
+const Label = styled(Text)`
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
```

Comment:
These rules seem to be non-standard:
https://caniuse.com/#search=line-clamp
https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient****
Apart from this, we should not use browser specific rules since babel / styled-components handle that, could we have an alternative style for lines 32-34?